src/test/groovy/net/vivin/gradle/versioning/VersionUtilsTests.groovy

```diff
@@ -0,0 +1,40 @@
+package net.vivin.gradle.versioning
+
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.internal.storage.file.FileRepository
+import org.eclipse.jgit.lib.Repository
+import org.gradle.testfixtures.ProjectBuilder
+import org.testng.annotations.BeforeMethod
+import org.testng.annotations.Test
+
+import static org.testng.Assert.assertFalse
+
+class VersionUtilsTests extends TestNGRepositoryTestCase {
+
+    @Override
```

Comment:
Is there a reason you overrode the setup method in `TestNGRepositoryTestCase`? I see that you create a temp directory and then clone into it. Was just curious as to why.